apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -99,6 +106,17 @@ AprilTagDetector::AprilTagDetector(ros::NodeHandle& nh, ros::NodeHandle& pnh) :
   pnh.param<float>("plane_angle_threshold", plane_angle_threshold_,0.0872665f);
   plane_angle_threshold_ = std::max(0.0f, std::min(90.0f, plane_angle_threshold_));
 
+
+  pnh.param<float>("tf_pose_acceptance_error_range", tf_pose_acceptance_error_range_, 4);
+  tf_pose_acceptance_error_range_ = std::max(0.0f, std::min(90.0f, tf_pose_acceptance_error_range_));
```

Comment:
Our other angle is in radians, and this is in degrees.  I'd prefer to keep everything in rad.